test/common/ops/hourglassPurchase.js

```diff
@@ -0,0 +1,138 @@
+import hourglassPurchase from '../../../common/script/ops/hourglassPurchase';
+import {
+  BadRequest,
+  NotAuthorized,
+} from '../../../common/script/libs/errors';
+import i18n from '../../../common/script/i18n';
+
+describe('user.ops.hourglassPurchase', () => {
+  let user;
+
+  beforeEach(() => {
+    user = {
```

Comment:
yeah, you can import generateUser from unit tests helpers and pass any change to the original object there